platform/pom.xml

```diff
@@ -178,7 +178,12 @@
             <dependency>
                 <groupId>net.minidev</groupId>
                 <artifactId>json-smart</artifactId>
-                <version>1.1.1</version>
+                <version>${json-smart.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>net.minidev</groupId>
+                <artifactId>asm</artifactId>
+                <version>1.0.2</version>
```

Comment:
Should we create a property for this dependency in the root pom like we did for the other ones?